common/locales/en/pets.json

```diff
@@ -1,10 +1,14 @@
 {
     "pets": "Pets",
+    "activePet": "Active Pet",
+    "noActivePet": "No Active Pet",
```

Comment:
It was, but I don't think so anymore
On Mon, Sep 12, 2016 at 9:28 PM Blade Barringer <notifications@github.com>
wrote:

> In common/locales/en/pets.json
> <https://github.com/HabitRPG/habitrpg/pull/7768#discussion_r78487740>:
>
> > @@ -1,10 +1,14 @@
> >  {
> >      "pets": "Pets",
> > +    "activePet": "Active Pet",
> > +    "noActivePet": "No Active Pet",
>
> Is this string used?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7768/files/72a6d73a9be23fd72ba98fbe74b2f14fbc840bfb#r78487740>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_87IdpqHBFVtpDKjqruQ55lOpIG4Sks5qpgo9gaJpZM4JIjd_>
> .
>